javascript/book.js

```diff
@@ -29,16 +50,14 @@ addUI();
 
 /* eslint-disable */
 const removeUI = (id) => {
-  booksArray = booksArray.filter((ele, index) => index !== id);
+  booksArray.removeBook(id);
+  //  booksArray = booksArray.filter((ele, index) => index !== id);
```

Comment:
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍🏻